### DISCUSSION TITLE
Timeframe selection and navigation in the dashboard

### DISCUSSION THREAD

**Discussion by Pindol83** [firefly-iii/firefly-iii#8593] [General]
It might be interesting to add, in the dashboard (top right), the option to choose the timeframe (day, week, month) with a dropdown menu, and arrows to navigate from the current week to the next, for example. The actual dropdown menu displaying the number of weeks can be a bit confusing. Essentially, integrating a selection similar to Google Calendar (desired timeframe and the ability to navigate forward and backward with arrows).

Let me know what you think.

Pindol

**Comment by JC5** [comment]
That's something I hope to add in the new layout, yes.
